tlf.1.in

```diff
@@ -663,20 +663,20 @@ Toggle between the CQ and S&P modes.
 Increase CW (Morse Code) speed (from call and exchange fields).
 .IP
 If the cursor is in the call input field and it is not empty and CHANGE_RST is
-set: increase his S value (the leftmost of the RST pair).
+set: increase his RST.
 .IP
 If the cursor is in the exchange field and it is not empty and CHANGE_RST is
-set: increase my S value (the rightmost of the RST pair).
+set: increase my RST.
 .
 .TP
 .B PgDown
 Decrease CW (Morse Code) speed (from call input and exchange fields).
 .IP
 If the cursor is in the call input field and it is not empty and CHANGE_RST is
-set: decrease his S value (the leftmost of the RST pair).
+set: decrease his RST.
 .IP
 If the cursor is in the exchange field and it is not empty and CHANGE_RST is
```

Comment:
`and CHANGE_RST is set` - I just found that if the patch applied, and `CHANGE_RST` **is NOT set**, the increase/decrease methods also modifies the sent/received values.

Reply:
Just tested again. If CHANGE_RST is NOT set PgUp PgDown changes only CW speed here. Please retest.

Reply:
Sorry - you're right. There was a forgotten `CHANGE_RST` keyword above in my config.

Sorry again.